marklogic-data-hub-central/ui/e2e/cypress/integration/persistence/persistence.spec.tsx

```diff
@@ -78,6 +80,41 @@ describe("Validate persistence across Hub Central", () => {
     cy.waitUntil(() => toolbar.getRunToolbarIcon()).click();
     cy.get("#personJSON .accordion-collapse").should("have.class", "accordion-collapse collapse show");
   });
+  it("should sort table by entityName asc", () => {
+    cy.waitUntil(() => toolbar.getModelToolbarIcon()).click();
+    modelPage.selectView("table");
+    entityTypeTable.getExpandEntityIcon("Customer");
+
+    modelPage.getSortIndicator().scrollIntoView().click();
+    modelPage.getEntityLabelNames().then(($els) => {
+      return (
+        Cypress.$.makeArray($els)
+          .map((el) => entityNamesAsc.push(el.innerText.toString()))
+      );
+    });
+  });
+
+  it("should be the same as the array", () => {
```

Comment:
Do you mind changing the description to something like "Validate that the table records on shown in the UI are sorted"